src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs

```diff
@@ -34,8 +34,9 @@ protected async Task<App> GetAppResource(string subscriptionId, string resourceG
                 Stamp = await GetHostingEnvironment(postBody.Stamp.Subscription, postBody.Stamp.ResourceGroup, postBody.Stamp != null ? postBody.Stamp.Name : string.Empty, postBody.Stamp, startTime, endTime, postBody.Kind),
                 AppType = GetApplicationType(postBody.Kind),
                 PlatformType = (!string.IsNullOrWhiteSpace(postBody.Kind) && postBody.Kind.ToLower().Contains("linux")) ? PlatformType.Linux : PlatformType.Windows,
-                StackType =  StackType.All //await this._siteService.GetApplicationStack(subscriptionId, resourceGroup, appName, (DataProviderContext)HttpContext.Items[HostConstants.DataProviderContextKey])
+                StackType =  StackType.All, //await this._siteService.GetApplicationStack(subscriptionId, resourceGroup, appName, (DataProviderContext)HttpContext.Items[HostConstants.DataProviderContextKey])
                 // Not Fetching Stack Information anymore for kusto optimizations. Instead, detectors should look to leverage a gist created for the same.
+                Tags = postBody.Tags
```

Comment:
Will requests on applens also contain the Tags information ?

Reply:
Also, we might want to check geomaster code to see if they are getting populated or not

Reply:
I tested applens locally and the tags were populated...

Sorry i am not following what else you want me to tes

Reply:
So this postBody object for  applens calls is populated by runtime host itself but for external calls, its populated by geomaster (since its already has the info) and sent as part of request. I was just wondering if we are populating this tags object in geomaster as well or not before sending the request. Most likely we will be doing it there as well but we might just want to quickly check the code there.  @rekhaswaminathan can help locating the code in geomaster which is doing this operation.

Reply:
I see....will wait for @rekhaswaminathan to respond.

So that will be in Antares code I guess

Reply:
@puneetg1983  @ShekharGupta1988 
GeoMaster is sending it as part of postbody:
https://msazure.visualstudio.com/One/_git/AAPT-Antares-Websites?path=%2Fsrc%2FHosting%2FAdministrationService%2FMicrosoft.Web.Hosting.Administration.GeoScale.Csm%2FControllers%2FDiagnosticCsmController.cs&version=GBdev&line=494&lineEnd=495&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
Perfect! Thank you so much @rekhaswaminathan